airflow/models/mappedoperator.py

```diff
@@ -383,6 +386,13 @@ def inherits_from_empty_operator(self) -> bool:
         """Implementing Operator."""
         return self._is_empty
 
+    @property
+    def reschedule(self) -> bool:
+        """Check if the operator is a sensor and has mode reschedule"""
+        if self._is_sensor:
+            return self.partial_kwargs['mode'] == 'reschedule'
+        raise AttributeError("reschedule is not defined for MappedOperator")
+
```

Comment:
@uranusjr What's your thought about this approach?

Reply:
Another option I'm exploring is changing the way the readytoreschedule dep is checked

Reply:
Would it be easier if we just make `reschedule` a separate attribute, and calculate that when a MappedOperator is created? This seem easier than adding a new `_is_sensor` that does not exist on BaseOperator.